client/style/calendar.css

```diff
@@ -0,0 +1,128 @@
+
+.reservation-popup-date-picker {
+  /* position: relative; */
+  width: 100%;
+  /* max-width: 33.33333%; */
+  /* width: 333px; */
+  margin: 0 auto;
+}
+
+.goog-date-picker {
+  position: absolute;
+  background: #fff;
+  border: 1px solid #ccc;
+  border-radius: 6px;
+  -webkit-box-shadow: 0 1px 2px rgba(10,10,10,0.2);
+  box-shadow: 0 1px 2px rgba(10,10,10,0.2);
+  text-align: center;
+  vertical-align: middle;
+  cursor: default;
+  z-index: 1054;
+  outline: none;
+  top: 35px;
+}
+
+.reservation-popup-date-picker .goog-date-picker-head td {
+  background: white;
+}
+
+.reservation-popup-date-picker .goog-date-picker-monthyear {
+  font-weight: bold;
+  padding: 15px;
+}
+
+.goog-date-picker td {
+  padding: 3px;
+}
+
+th, td {
+  vertical-align: middle;
+}
+
+.goog-date-picker button {
+  margin: 0;
+  padding: 3px 0;
+  outline: 0;
+  border: none;
+  color: #666;
+  font-weight: bold;
+  background: transparent;
+  cursor: pointer;
+}
+
+.goog-date-picker th {
+  color: #d32323;
+  font-size: 12px;
+  font-weight: bold;
+  padding: 3px;
+  text-align: center;
+}
+
+.reservation-popup-date-picker .goog-date-picker-date:last-child {
+  border-right: none;
+}
+
+.reservation-popup-date-picker .goog-date-picker-disabled {
+  color: #e6e6e6;
+}
+
+.reservation-popup-date-picker .goog-date-picker-date {
+  width: auto;
+  line-height: 20px;
+  border: 1px solid #ccc;
+  border-bottom: none;
+  border-left: none;
+  padding: 10px;
+  vertical-align: middle;
+  text-align: center;
+}
+
+.goog-date-picker td {
+  padding: 3px;
+}
+
+.reservation-popup-date-picker .goog-date-picker tbody th:not(.goog-date-picker-wday) {
+  display: none;
+}
+
+.reservation-popup-date-picker .drop-menu-arrow {
+  top: 13px;
+  z-index: 1055;
+  text-align: center;
+}
+
+.drop-menu-arrow {
+  position: relative;
+  top: -12px;
+}
+
+.reservation-popup-date-picker .drop-menu-arrow::before, .reservation-popup-date-picker .drop-menu-arrow::after {
+  right: auto;
+}
+
+.drop-menu-arrow::after, .drop-menu-arrow::before {
+  bottom: 100%;
+  right: 8px;
+  border: solid transparent;
+  content: "";
+  height: 0;
+  width: 0;
+  position: absolute;
+  pointer-events: none;
+}
+
+.drop-menu-arrow::after {
+  border-bottom-color: white;
+  border-width: 10px;
+  margin-left: -10px;
+}
+
+.drop-menu-arrow::before {
+  border-bottom-color: rgba(0,0,0,0.2);
+  border-width: 11px;
+  margin-left: -11px;
+}
+.reservation-popup-date-picker .drop-menu-arrow::before, .reservation-popup-date-picker .drop-menu-arrow::after {
+  right: auto;
+}
+
```

Comment:
Daniel's CR stopping here.

Reply:
Looks pretty good. But, so many CSS selectors, some of which seem necessary, and others that might be extraneous. You should use a standard naming convention, especially for your CSS. And, the naming should be clearer semantically and more human readable.